lib/msal-browser/src/config/Configuration.ts

```diff
@@ -340,6 +341,14 @@ export function buildConfiguration(
             throw ClientConfigurationError.createCannotAllowNativeBrokerError();
     }
 
+    // Throw an error if using a known Microsoft authority with OIDC compliance mode
+    if(userInputAuth?.authority && userInputAuth?.protocolMode === ProtocolMode.OIDC) {
+        const knownMSAuthorities = Object.keys(rawMetdataJSON.endpointMetadata);
```

Comment:
nit: `rawMetadataJSON`?

Reply:
I noticed that too, the original variable is actually called that and I wasn't sure why, I can change it if there isn't a reason for it.

Reply:
For some reason this change caused many failing tests. Because the typo appears at many points in the code, I assume it's purposeful.

Reply:
You can import the named `EndpointMetadata` variable from `AuthorityMetadata` instead. It's called `rawMetadataJson` because it is a "raw" copy-paste from the network response.

These are the last couple lines in AuthorityMetadata:
```typescript
export const EndpointMetadata = rawMetdataJSON.endpointMetadata;
export const InstanceDiscoveryMetadata =
    rawMetdataJSON.instanceDiscoveryMetadata;
```